examples/stable_diffusion/convert_sd_adapter_to_peft.py

```diff
@@ -297,6 +312,18 @@ def detect_adapter_type(keys: List[str]) -> PeftType:
     # Process each model sequentially
     for model, model_name in [(text_encoder, "text_encoder"), (unet, "unet")]:
         config = construct_config_fn(adapter_info[model_name])
+
+        # Output warning for LoHa with use_effective_conv2d
+        if (
+            isinstance(config, LoHaConfig)
+            and getattr(config, "use_effective_conv2d", False)
+            and args.loha_weights_fix is False
+        ):
+            logging.warning(
+                'Current official LoHa implementation contains a bug, which can be fixed with "--loha_weights_fix".\n'
```

Comment:
Again, let's avoid words like "Current".

Reply:
Okay, I replaced it with LyCORIS library version description.